ibis/util.py

```diff
@@ -726,3 +726,17 @@ def __ne__(self, other):
             return self.obj != other.obj
         else:
             return NotImplemented
+
+
+# TODO: remove me in 10.0
+def _warn_schema():
+    warn_deprecated(
+        name="schema",
+        as_of="9.0",
+        removed_in="10.0",
+        instead="Use the `database` kwarg with one of the following patterns:"
+        '\ndatabase="database"'
+        '\ndatabase=("catalog", "database")'
+        '\ndatabase="catalog.database"',
+        # TODO: add option for namespace object
+    )
```

Comment:
I'm going to move this into the deprecation class helper mixin, but after I squash down everything else because that rebase might be a bit gnarly.